libpod/pod.go

```diff
@@ -70,6 +72,14 @@ type PodConfig struct {
 
 	// ID of the pod's lock
 	LockID uint32 `json:"lockID"`
+
+	//ResourceLimits contains linux specific CPU data for the pod
+	ResourceLimits *spec.LinuxResources `json:"resource_limits,omitempty"`
```

Comment:
This should be in InfraConfig

Reply:
in addition to podConfig, correct? if I remove this it breaks pretty much everything i've written. PodConfig uses this to pass to InfraConfig down the line.

Reply:
Why do we need two copies? When we do this for other infra container fields, we store a single copy in `InfraConfig` - duplicating this in two places in the Pod structure in the DB seems unnecessary?

Reply:
Yep you're right, just  had to change my options.go code to populate the infra fields instead.